docs/reference/treyresearch/armTemplates/auxiliary/diagnosticsAndSecurity.json

```diff
@@ -129,7 +143,7 @@
         "policyDefinitions": {
             "deployAzureActivityLog": "[concat('/providers/Microsoft.Management/managementGroups/', parameters('topLevelManagementGroupPrefix'), '/providers/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-ActivityLog')]",
```

Comment:
Can you pls update the policies.json from Wingtip, and change the 'deployAzureActivityLog' to point towards the built-in policy? (This recently moved from custom to built-in, so we are not maintaining it anymore from here)

Reply:
@krnese , updating these changes and running a local test. I will update you once the test is complete.